packages/host/tests/integration/components/ai-assistant-panel-test.gts

```diff
@@ -1351,157 +1344,6 @@ module('Integration | ai-assistant-panel', function (hooks) {
-        msgtype: 'org.boxel.message',
-        formatted_body: 'What is a french bulldog?',
-        format: 'org.matrix.custom.html',
-      },
-      origin_server_ts: Date.now() - 98,
-      unsigned: {
-        age: 105,
-        transaction_id: '1',
-      },
-      status: null,
-    });
-
-    await addRoomEvent(matrixService, {
-      event_id: 'event3',
-      room_id: roomId,
-      state_key: 'state',
-      type: 'm.room.message',
-      sender: '@aibot:localhost',
-      content: {
-        body: 'French bulldog is a',
-        msgtype: 'm.text',
-        formatted_body: 'French bulldog is a',
-        format: 'org.matrix.custom.html',
-      },
-      origin_server_ts: Date.now() - 97,
-      unsigned: {
-        age: 105,
-        transaction_id: '1',
-      },
-      status: null,
-    });
-
-    await waitFor('[data-test-message-idx="3"]');
-
-    assert
-      .dom('[data-test-message-idx="1"] [data-test-ai-avatar]')
-      .doesNotHaveClass(
-        'ai-avatar-animated',
-        'Answer to my previous question is not in progress',
-      );
-    assert
-      .dom('[data-test-message-idx="3"] [data-test-ai-avatar]')
-      .hasClass(
-        'ai-avatar-animated',
-        'Answer to my current question is in progress',
-      );
-
-    await click('[data-test-past-sessions-button]');
-    assert
-      .dom("[data-test-enter-room='New AI Assistant Chat']")
-      .includesText('Thinking');
-    assert.dom('[data-test-is-streaming]').exists();
-
-    await addRoomEvent(matrixService, {
-      event_id: 'event4',
-      room_id: roomId,
-      state_key: 'state',
-      type: 'm.room.message',
-      sender: '@aibot:localhost',
-      content: {
-        body: 'French bulldog is a French breed of companion dog or toy dog.',
-        msgtype: 'm.text',
-        formatted_body:
-          'French bulldog is a French breed of companion dog or toy dog',
-        format: 'org.matrix.custom.html',
-        isStreamingFinished: true, // This is an indicator from the ai bot that the message is finalized and the openai is done streaming
-        'm.relates_to': {
-          rel_type: 'm.replace',
-          event_id: 'event3',
-        },
-      },
-      origin_server_ts: Date.now() - 96,
-      unsigned: {
-        age: 105,
-        transaction_id: '1',
-      },
-      status: null,
-    });
-
-    await waitFor('[data-test-message-idx="3"]');
-    assert
-      .dom('[data-test-message-idx="1"] [data-test-ai-avatar]')
-      .doesNotHaveClass(
-        'ai-avatar-animated',
-        'Answer to my previous question is not in progress',
-      );
-    assert
-      .dom('[data-test-message-idx="3"] [data-test-ai-avatar]')
-      .doesNotHaveClass(
-        'ai-avatar-animated',
-        'Answer to my last question is not in progress',
-      );
-
-    assert
-      .dom("[data-test-enter-room='New AI Assistant Chat']")
-      .doesNotContainText('Thinking');
-    assert.dom('[data-test-is-streaming]').doesNotExist();
-  });
```

Comment:
Are the assertions here covered elsewhere?

Reply:
Yes this is a redundant test with duplicated code

Reply:
https://github.com/cardstack/boxel/pull/1407/commits/f1a346a028bb88e25ef0a952d86a22650e2779bd I did to use the newly introduced renderAiAsssistantPanel